packages/dom/test/unit/lib/ElementAssertion.test.tsx

```diff
@@ -12,6 +12,12 @@ function TestComponent(): ReactElement {
   );
 }
 
+const TestComponentElement = (): ReactElement => (
+  <button role="button" type="submit" className="btn primary" disabled>
+    click me
+  </button>
+);
```

Comment:
In general, it's better to create react components as named functions.  Arrow functions will lose their name at runtime, but named functions won't. That way, the component's `displayName` is the same as the function's name.
```suggestion
function TestComponentElement(): ReactElement {
  return (
    <button role="button" type="submit" className="btn primary" disabled>
      click me
    </button>
  );
}
```

Reply:
Thanks @JoseLion , I just addressed all your comments. Feel free to take a look at it again.